service/src/chain_spec.rs

```diff
@@ -54,7 +54,7 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 	const DAYS: u64 = HOURS * 24;
 	GenesisConfig {
 		consensus: Some(ConsensusConfig {
-			code: include_bytes!("../../runtime/wasm/target/wasm32-unknown-unknown/release/polkadot_runtime.compact.wasm").to_vec(),	// TODO change
+			code: include_bytes!("../../runtime/wasm/target/wasm32-unknown-unknown/release/polkadot_runtime.compact.wasm").to_vec(),
```

Comment:
Why did you remove the `TODO`?
This is as well a substrate issue: https://github.com/paritytech/substrate/blob/master/node/cli/src/chain_spec.rs#L61